spec/endtoend/endtoend_spec.rb

```diff
@@ -144,15 +144,7 @@
       expect(@workspace.tsuru_command.stdout).to include query
     end
 
-    it "should be able to connect to the applitation via HTTPS with a valid cert" do
```

Comment:
np. I think that it too would have benefited from being a separate commit saying that, then I wouldn't have needed to ask. But not a blocker for merging.